src/Altinn.PSWin.Receiver/Exceptions/DrParserException.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace LinkMobility.PSWin.Receiver.Exceptions
+{
+
+    [System.Serializable]
```

Comment:
Can probably remove this attribute

Reply:
copied directly from Links library. not keen to make any changes to the code if it builds and runs. We are replacing it in the next few days hopefully